src/api/admin.js

```diff
@@ -0,0 +1,19 @@
+import { api } from "./index";
+
+export const fetchCustomers = async () => {
+  const response = await api.get("/admin/customers");
+  return response.data;
+};
+
+export const activateUser = async (id) => {
+  await api.post(`/admin/users/${id}/activate`);
+};
+
+export const deactivateUser = async (id) => {
+  await api.post(`/admin/users/${id}/deactivate`);
+};
+
+export const fetchProperties = async () => {
+  const response = await api.get("/public/properties");
```

Comment:
I don't think admin will use this endpoint, also check the api/properties file it has that fetch already, no need to duplicate it